src/DataCollector/RedisDataCollector.php

```diff
@@ -76,7 +76,7 @@ public function getTotalExecutionTime(): float
             $time += $value['executiontime'];
 
             return $time;
-        });
+        }) ?? 0;
```

Comment:
I'd have said:

```suggestion
        }) ?? 0.0;
```

but I'm surprised that `0` also works fine while there is `declare(strict_types=1);` at the top of the file 🤔 

```php
<?php

declare(strict_types=1);

function test(): float
{
    return 0;
}

var_dump(test());
```

produces `float(0)` 🤷‍♂